### ISSUE TITLE
softmax with opset8 is not supported

### ISSUE THREAD

**Issue by alex-ht** [openvinotoolkit/openvino_contrib#397]
Hi, I'm trying to convert a keras model which has CNN2d + Softmax layer at output.
It works fine on Intel CPU, but I got an error at arm64 platform
```sh
[02:42:00.6018]I[plugin.cpp:346] [AUTOPLUGIN]:device:CPU, priority:0
[02:42:00.6021]I[executable_network.cpp:163] [AUTOPLUGIN]ExecutableNetwork start
[02:42:00.6026]I[executable_network.cpp:183] [AUTOPLUGIN]:select device:CPU
Arm Plugin: Nodes from model_1 are not supported by plugin:
	conv2d_25/truediv:0 (Softmax.0)
```
Version: 
- releases/2022/1
- Dockerfile.RPi64_focal

Model:
- [xml](https://drive.google.com/file/d/1pH2mRuCEHFDJxg0VGug2oP46ooumKu6f/view?usp=sharing)

**Comment by alex-ht** [comment]
I found a workaround!
The original xml has a `version="opset8"` attribute and `<data axis="-1"/>`like this
```xml
<layer id="190" name="conv2d_25/truediv:0" type="SoftMax" version="opset8">
			<data axis="-1"/>
			<rt_info>
				<attribute name="fused_names" version="0" value="conv2d_25/truediv:0"/>
			</rt_info>
			<input>
				<port id="0" precision="FP32">
					<dim>1</dim>
					<dim>16</dim>
					<dim>16</dim>
					<dim>2</dim>
				</port>
			</input>
			<output>
				<port id="1" precision="FP32" names="conv2d_25/truediv:0">
					<dim>1</dim>
					<dim>16</dim>
					<dim>16</dim>
					<dim>2</dim>
				</port>
			</output>
		</layer>
```
change it to `"opset1"` and give `axis` a positive index value.
```xml
<layer id="190" name="conv2d_25/truediv:0" type="SoftMax" version="opset1">
                        <data axis="3"/>
```
